sim/target-agnostic.mk

```diff
@@ -58,10 +58,10 @@ $(GENERATED_DIR):
 
 # empty recipe to help make understand multiple targets come from single recipe invocation
 # without using the new (4.3) '&:' grouped targets see https://stackoverflow.com/a/41710495
-$(VERILOG) $(HEADER) $(fame_annos): $(VERILOG).intermediate
+$(VERILOG) $(HEADER) $(fame_annos): $(VERILOG).intermediate ;
```

Comment:
This rule needs the semicolon to make it explicitly empty (without requiring whitespace or dumping extraneous prints during runtime)